### PR TITLE
fix(frontend): tag prefix for vnext

### PR DESCRIPTION
**Pull request by joseph118**
prefix tag with `-vnext` when checking on `vnext` branch

### CODE REVIEW

**Comment by joseph118** [comment]
vnext;
image


master;
image

File: .github/workflows/frontend-build-and-publish-apps.yml [hunk 1]
```diff
@@ -161,7 +161,7 @@ jobs:
         uses: mukunku/tag-exists-action@v1.0.0
         id: tag
         with:
-          tag: ${{ env.APP_VERSION }}
+          tag: "${{ env.APP_VERSION }}${{ steps.branch.outputs.name == 'vnext' && '-vnext' || '' }}"
```
Comment (kurtaqui) [inline .github/workflows/frontend-build-and-publish-apps.yml:164]: this check can go inside the step above `Set environment variables` and you can remove the duplicate code from git tag release

**Comment by joseph118** [comment]
before merging this in, we need to align cashier vnext

**Comment by joseph118** [comment]
master;
image

vnext;
image

**Review by kurtaqui — APPROVED** [review]

**Pull request merged**